cmake/Modules/UseDamaris.cmake

```diff
@@ -0,0 +1,32 @@
+if(USE_DAMARIS_LIB)
+  if (Damaris_FOUND)
+    set(HAVE_DAMARIS 1)
+    message(STATUS "The Damaris library was found: ${Damaris_VERSION} ${Damaris_DIR}")
+    if (Damaris_HAS_HDF5)
+      message(STATUS "The Damaris library has HDF5 support: ${HDF5_VERSION}")
+    else()
+      message(STATUS "The Damaris library does NOT have HDF5 support")
+    endif()
+    if (Damaris_HAS_VISIT)
+      message(STATUS "The Damaris library has VisIt support: ${VisIt_VERSION}")
+    else()
+      message(STATUS "The Damaris library does NOT have VisIt support")
+    endif()
+    if (Damaris_HAS_CATALYST)
+      message(STATUS "The Damaris library has Catalyst support: ${Catalyst_VERSION}  ${Paraview_VERSION}")
+    else()
+      message(STATUS "The Damaris library does NOT have Catalyst support")
+    endif()
+  else()
+    message(STATUS "The Damaris library was requested but NOT found")
+  endif()
+else()  # User did not request Damaris support
+  if (Damaris_FOUND)
+    message(STATUS "The Damaris library was NOT requested but was found: ${Damaris_VERSION} ${Damaris_DIR}")
+  else()
+    message(STATUS "The Damaris library was NOT requested and was NOT found")
+  endif()
+  unset(HAVE_DAMARIS)
+endif()
+
+mark_as_advanced(HAVE_DAMARIS)
```

Comment:
Naive question here: Is it not the option name (USE_DAMARIS_LIB) that should be marked as advanced, not the name of the define for the config? (Assuming "advanced" here refers to the toggle in ccmake or equivalent.)

Reply:
advanced means it's hidden in ccmake.